templates/emails/vendor-new-order.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * New Order Email.
+ *
+ * An email sent to the vendor when a new order is created by customer.
+ *
+ * @class       VendorNewOrder
+ * @version     2.6.8
+ *
+ */
+
+ if ( ! defined( 'ABSPATH' ) ) {
+    exit;
+ }
+
+$text_align = is_rtl() ? 'right' : 'left';
+
+ /**
+  * @hooked WC_Emails::email_header() Output the email header
+  */
+ do_action( 'woocommerce_email_header', $email_heading, $email );
+
+ $order_url = dokan_get_navigation_url( 'orders?order_id='.$order->get_order_number().'&_view_mode=email&permission=1' );
```

Comment:
Please use `add_query_args` function for rendering url and must use `esc_url` function for escaping